File: odd_even.java
public class OddEvenCheck {
    public static String oddOrEven(int num) {
        return (num % 2 == 0) ? "Even" : "Odd";
    }

    public static void main(String[] args) {
        System.out.println(oddOrEven(10)); // Output: Even
        System.out.println(oddOrEven(7));  // Output: Odd
    }
}

File: odd_even.cs
using System;

class Program {
    static string OddOrEven(int num) {
        return (num % 2 == 0) ? "Even" : "Odd";
    }

    static void Main() {
        Console.WriteLine(OddOrEven(10)); // Output: Even
        Console.WriteLine(OddOrEven(7));  // Output: Odd
    }
}
